### PR TITLE
Rename entrypoint to init; formatting

### PR DESCRIPTION
**Pull request by lucekdudek**

### CODE REVIEW

File: gaps/gap-16_golem_compose/gaom.schema.json [hunk 1]
```diff
@@ -1,153 +1,152 @@
...
-          },
-          "author": {
-            "type": "string",
-            "description": "Author"
-          },
-          "version": {
-            "type": "string",
-            "description": "Version of the application (following the sem-ver convention)"
-          }
+  "$id": "https://golem.network/gaom.schema.json",
```
Comment (shadeofblue) [inline gaps/gap-16_golem_compose/gaom.schema.json:2]: uhh - why is such a large update needed?... please don't do this

**Review by shadeofblue — CHANGES_REQUESTED** [review]
I don't think such a minimal change justifies a reformat of the whole `json` file...

**Comment by shadeofblue** [comment]
@lucekdudek I usually have no problem with having files aligned to a common format ... but in this case I believe it's taken one step too far... please update the documents just where an update is needed and refrain from bulk updates just for the sake of your particular editor setup, thank you :bowing_man:

**Comment by lucekdudek** [comment]
> @lucekdudek I usually have no problem with having files aligned to a common format ... but in this case I believe it's taken one step too far... please update the documents just where an update is needed and refrain from bulk updates just for the sake of your particular editor setup, thank you bowing_man

Fixed

**Review by shadeofblue — APPROVED** [review]

**Comment by stranger80** [comment]
This PR is missing an update to gaom schema entity diagram.
Let me add this as well.

**Pull request merged**